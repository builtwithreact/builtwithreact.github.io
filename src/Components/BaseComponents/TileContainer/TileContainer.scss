.TileContainer {
  display: flex;
  justify-content: space-between;
  margin: 25px 0;
  .Tile {
  }
}