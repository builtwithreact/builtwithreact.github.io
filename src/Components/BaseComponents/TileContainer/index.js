import React, { Component } from 'react';
//import PropTypes from 'prop-types';
//import classnames from 'classnames';
import _map from 'lodash/map';
import Tile from './../Tile';
import Sample from './Sample.png'
import './TileContainer.css';

const propTypes = {}
const defaultProps = {}

export default class TileContainer extends Component {
    render() {
        return (
            <div className="TileContainer">
                <div className="Tile">
                    <div className="Tile-Image">
                        {/* <img src="http://madewithreact.com/content/images/2017/03/react-logo-generator-1.png" /> */}
                        <img src={Sample} alt=""/>
                    </div>
                    <div className="Tile-Details">
                        <hr className="divider" />
                        <div className="ProjectInfo">
                            <h3 className="ProjectInfo-Title"> React Logo Generator </h3>
                            <p className="ProjectInfo-Description">This is Play Area to Change to generate React Logo with different colors</p>
                        </div>
                        <div className="AuthorInfo">
                            <a href="https://github.com/pankajladhar" target="_blank">
                                {/* <img className="AuthorInfo-Avatar" src="https://avatars2.githubusercontent.com/u/6375735?v=4" name="Pankaj Ladhar" /> */}
                                <img className="AuthorInfo-Avatar" src={Sample} alt=""/>
                            </a>
                            <div>
                                <a href="https://github.com/pankajladhar" target="_blank">Pankaj Ladhar</a>
                                <span className="AuthorInfo-submissionDate">2017 Sept 15</span>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        );
    }
}

TileContainer.propTypes = propTypes;
TileContainer.defaultProps = defaultProps;
