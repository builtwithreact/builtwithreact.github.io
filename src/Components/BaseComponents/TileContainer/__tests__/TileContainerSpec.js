import React from 'react';
import renderer from 'react-test-renderer';
import TileContainer from './../index';
jest.mock('./../../Tile', () => 'Tile')

describe('<TileContainer />', () => {
    it('should render correctly', () => {
        let tree = renderer.create(
            <TileContainer />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
