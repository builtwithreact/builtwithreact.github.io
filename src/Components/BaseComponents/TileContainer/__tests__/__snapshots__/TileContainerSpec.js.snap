// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TileContainer /> should render correctly 1`] = `
<div
  className="TileContainer"
>
  <div
    className="Tile"
  >
    <div
      className="Tile-Image"
    >
      <img
        alt=""
        src="Sample.png"
      />
    </div>
    <div
      className="Tile-Details"
    >
      <hr
        className="divider"
      />
      <div
        className="ProjectInfo"
      >
        <h3
          className="ProjectInfo-Title"
        >
           React Logo Generator 
        </h3>
        <p
          className="ProjectInfo-Description"
        >
          This is Play Area to Change to generate React Logo with different colors
        </p>
      </div>
      <div
        className="AuthorInfo"
      >
        <a
          href="https://github.com/pankajladhar"
          target="_blank"
        >
          <img
            alt=""
            className="AuthorInfo-Avatar"
            src="Sample.png"
          />
        </a>
        <div>
          <a
            href="https://github.com/pankajladhar"
            target="_blank"
          >
            Pankaj Ladhar
          </a>
          <span
            className="AuthorInfo-submissionDate"
          >
            2017 Sept 15
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`TileContainer renders correctly 1`] = `
<div
  className="TileContainer"
>
  <div
    className="Tile"
  >
    <div
      className="Tile-Image"
    >
      <img
        alt=""
        src="Sample.png"
      />
    </div>
    <div
      className="Tile-Details"
    >
      <hr
        className="divider"
      />
      <div
        className="ProjectInfo"
      >
        <h3
          className="ProjectInfo-Title"
        >
           React Logo Generator 
        </h3>
        <p
          className="ProjectInfo-Description"
        >
          This is Play Area to Change to generate React Logo with different colors
        </p>
      </div>
      <div
        className="AuthorInfo"
      >
        <a
          href="https://github.com/pankajladhar"
          target="_blank"
        >
          <img
            alt=""
            className="AuthorInfo-Avatar"
            src="Sample.png"
          />
        </a>
        <div>
          <a
            href="https://github.com/pankajladhar"
            target="_blank"
          >
            Pankaj Ladhar
          </a>
          <span
            className="AuthorInfo-submissionDate"
          >
            2017 Sept 15
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
